packages/cyberstorm/src/components/componentStyles/PackageFlag.module.css

```diff
@@ -0,0 +1,29 @@
+.root {
+  display: flex;
+  flex-direction: row;
+  gap: var(--gap-3xs);
+  align-items: center;
+  justify-content: center;
+
+  padding: var(--padding-4xs) var(--padding-3xs);
+  border: var(--size-6xs) solid var(--border-color);
+  border-radius: 0 var(--border-radius) var(--border-radius) 0;
+  color: var(--color-cyber-green);
+  background-color: var(--color-surface--3);
+
+  --border-color: transparent;
+  --border-radius: var(--border-radius-5xs);
+}
+
+.label {
+  font: var(--font-inter);
+  font-weight: var(--font-weight-extra-bold);
+  font-size: var(--size-xs);
```

Comment:
For simplicity's sake and to be consistent with other similar instances, and since these seem to be using `em`, declaring font-size for the root only instead of icon and label separately is preferred so one can change both with one rule. In the unlikely event one wants to change he proportions of the two items one can always add an overriding rule to either one of them.

Reply:
Aight, changed it to be declared on `.root`.